html/pages/devices.inc.php

```diff
@@ -210,264 +197,254 @@
-    }
-}
+    echo '
+                    <th data-column-id="hardware">Platform</th>
+                    <th data-column-id="os">Operating System</th>
+                    <th data-column-id="uptime">Uptime</th>
+    ';
 
-?>
-                      "</select>"+
-                      "</div>"+
-                      "<button type=\"submit\" class=\"btn btn-default input-sm\">Search</button>"+
-                      "<div class=\"form-group\">"+
-                      "<a href=\"<?php echo(generate_url($vars)); ?>\" title=\"Update the browser URL to reflect the search criteria.\" >&nbsp;Update URL</a> |"+
-                      "<a href=\"<?php echo(generate_url(array('page' => 'devices', 'section' => $vars['section'], 'bare' => $vars['bare']))); ?>\" title=\"Reset critera to default.\" >&nbsp;Reset</a>"+
-                      "</div>"+
-                      "</form></span></div>"+
-                      "<div class=\"col-sm-3 actionBar\"><p class=\"{{css.actions}}\"></p></div></div></div>"
+    if ($subformat == "detail") {
+        echo '<th data-column-id="location">Location</th>';
+    }
 
-<?php
-if (isset($vars['searchbar']) && $vars['searchbar'] == "hide") {
-    echo "searchbar = ''";
-}
-?>
-
-var grid = $("#devices").bootgrid({
-    ajax: true,
-    rowCount: [50,100,250,-1],
-    columnSelection: false,
-    formatters: {
-        "status": function(column,row) {
-            return "<h4><span class='label label-"+row.extra+" threeqtr-width'>" + row.msg + "</span></h4>";
-        }
-    },
-    templates: {
-        header: searchbar
-    },
-    post: function ()
-    {
-        return {
-            id: "devices",
-            format: '<?php echo mres($vars['format']); ?>',
-            hostname: '<?php echo htmlspecialchars($vars['hostname']); ?>',
-            os: '<?php echo mres($vars['os']); ?>',
-            version: '<?php echo mres($vars['version']); ?>',
-            hardware: '<?php echo mres($vars['hardware']); ?>',
-            features: '<?php echo mres($vars['features']); ?>',
-            location: '<?php echo mres($vars['location']); ?>',
-            type: '<?php echo mres($vars['type']); ?>',
-            state: '<?php echo mres($vars['state']); ?>',
-            disabled: '<?php echo mres($vars['disabled']); ?>',
-            ignore: '<?php echo mres($vars['ignore']); ?>',
-            group: '<?php echo mres($vars['group']); ?>',
-        };
-    },
-    url: "ajax_table.php"
-});
-
-</script>
+    echo '
+                    <th data-column-id="actions" data-sortable="false" data-searchable="false">Actions</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+    ';
 
-<?php
+    ?>
+
+    <script>
+        var grid = $("#devices").bootgrid({
+            ajax: true,
+            rowCount: [5, 10, 50, 100, 250, -1],
+            columnSelection: false,
+            formatters: {
+                "status": function (column, row) {
```

Comment:
This is still showing a bigger box with small text - I think you just need to revert the status box back imho.